packages/account/src/Containers/Account/account.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { FadeWrapper, Loading } from '@deriv/components';
+import { matchRoute, routes as shared_routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { flatten } from '../../Helpers/flatten';
+import PageOverlayWrapper from './page-overlay-wrapper';
+import { TRoute } from '../../Types';
+import 'Styles/account.scss';
+
+type TAccountProps = RouteComponentProps & {
+    routes: Array<TRoute>;
+};
+
+/**
+ * Component that renders the account section
+ * @name Account
+ * @param history - history object passed from react-router-dom
+ * @param location - location object passed from react-router-dom
+ * @param routes - routes object passed from react-router-dom
+ * @returns React component
+ */
+const Account = observer(({ history, location, routes }: TAccountProps) => {
+    const { client, ui } = useStore();
+    const {
+        is_virtual,
+        is_logged_in,
+        is_logging_in,
+        is_pending_proof_of_ownership,
+        landing_company_shortcode,
+        should_allow_authentication,
+    } = client;
+    const { toggleAccountSettings, is_account_settings_visible } = ui;
+    const subroutes = flatten(routes.map(i => i.subroutes));
```

Comment:
can you check please, maybe we can get rid of this function? it is used only  in this file.
map already returns new array, why do we need to extract it again? looks like unnecessary action